src/driver/predefs.c

```diff
@@ -453,6 +453,8 @@ void add_predefined_macros(void)
 
 	/* TODO: __CHAR16_TYPE__, __CHAR32_TYPE__ */
 
+    define_special_macro_functions();
```

Comment:
I think this needs to be indented with tabs (unless this is a display problem in github)